.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: rust
 rust:
+  - stable
   - nightly
 script:
-  - cargo test
+  - cargo build
+  - |
+    [ $TRAVIS_RUST_VERSION != nightly ] ||
+    cargo test
```

Comment:
re: inverting the condition, that leaves the command with a non-zero exit status, which will be read by travis as a failure.